QSB/QSB.cs

```diff
@@ -33,6 +35,8 @@ private void Start()
 
             GlobalMessenger.AddListener(EventNames.RestartTimeLoop, OnLoopStart);
             GlobalMessenger.AddListener(EventNames.WakeUp, OnWakeUp);
+
+            SceneManager.sceneLoaded += OnSceneLoad;
```

Comment:
We use LoadManager elsewhere. It's generally better. Won't change anything here, but consistency ;)